haskell/cabal.bzl

```diff
@@ -581,6 +615,10 @@ stack_snapshot = repository_rule(
         "deps": attr.label_list(
             doc = "Dependencies of the package set, e.g. system libraries or C/C++ libraries.",
         ),
+        "tools": attr.label_list(
+            doc = """Tool dependencies. They are built using the host configuration, since
```

Comment:
I don't see the host configuration enforced anywhere. [`label_list`](https://docs.bazel.build/versions/master/skylark/lib/attr.html#label_list) seems to suppor the `cfg = "host"` attribute, so, maybe that should be set?

Reply:
Fixed.